javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+var h1 = document.querySelector('h1');
+
+function countdown() {
+  if (h1.textContent === '4') {
```

Comment:
This is a very manual approach. How can you do this where you don't need a condition for every number? What if we wanted to count down from 1000? That would be horrifying! Consider that you can have variables that change in value!